npm/test.js

```diff
@@ -3,8 +3,26 @@ require('shelljs/global');
 require('colors');
 
 var prettyms = require('pretty-ms'),
+    mocks = require('./simple-mock'),
     startedAt = Date.now();
 
+
+mocks.mockGETrequest();
```

Comment:
This section will have to be changed if more nocks are added down the line, update the nock helper structure to introduce all these mocks in one go.